docs/DeveloperGuide.md

```diff
@@ -336,6 +336,30 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+
+    * 3a1. FAKEJARVIS shows an error message.
+
+      Use case resumes at step 2.
+
+
+**Use case: Grade an assignment**
+
+**MSS**
```

Comment:
Since this is in the context of a TA helper, 
Might be better to rename "persons" to "students"

Reply:
I used "persons" to standardize with the above 2 use cases. If we want to rename it to "students", then we should probably standardize it for all other use cases too.